DoxygenRule.cmake

```diff
@@ -88,6 +88,19 @@ if(NOT COMMON_ORGANIZATION_NAME)
   set(COMMON_ORGANIZATION_NAME Unknown)
 endif()
 
+# collect application help page
+get_property(__help GLOBAL PROPERTY ${PROJECT_NAME}_HELP)
```

Comment:
That's a fair question. My take on this is to try to document every in/out variable of each CMake file, because it makes the interdependence between different files explicit. Having hidden global variables is not good for maintenance in the long run. Regarding this specific case, I am not sure if there is a reason users should be discouraged to add their own help file for a python application for instance (tide, rtneuron-app.py, ...)